pkg/api/multiarchbuildconfig/v1/types.go

```diff
@@ -14,6 +15,9 @@ import (
 	buildv1 "github.com/openshift/api/build/v1"
 )
 
+//go:embed ci.openshift.io_multiarchbuildconfigs.yaml
```

Comment:
I would move this into its own file at `/pkg/api/clusterinit.go`. IMHO this is a good move that would anticipate future needs when we might want to add more CRDs or stuff. By having a `//go:embed` directive rooted at `/pkg/api` means we can reference everything in `/pkg/api/**`.